Sources/Target/Windows/Process.cpp

```diff
@@ -104,10 +122,52 @@ ErrorCode Process::detach() {
   return kSuccess;
 }
 
+BOOL Process::writeDebugBreakCode(uint64_t address) {
+  FARPROC exitThreadAddress =
+      GetProcAddress(GetModuleHandleA("ntdll"), "RtlExitUserThread");
+  if (exitThreadAddress == NULL) {
+    return FALSE;
+  }
+
+  ByteVector codestr(&debugBreakCode[0],
+                     &debugBreakCode[sizeof(debugBreakCode)]);
+  *reinterpret_cast<uint32_t *>(&codestr[kRtlExitUserThreadOffset]) =
```

Comment:
This code assumes the endianness of ds2 and the endianness of the inferior are the same. I think this is a safe assumption.